### PR TITLE
Add method to get all dashboards for a given user

### PR DESCRIPTION
**Pull request by j-ma-sf**
This PR adds an endpoint for getting ALL dashboards for an owner.
Also updates the version.

### CODE REVIEW

**Comment by j-ma-sf** [comment]
@haridsv

File: argusclient/client.py [hunk 1]
```diff
@@ -377,6 +377,14 @@ def get_user_dashboard(self, ownerName, dashboardName, shared=True):
...
             return dashboards[0]
 
+    def get_user_dashboards(self, ownerName, shared=True, limit=None, version=None):
+        """
+        Gets dashboards owned by ownerName.
+
+        :return: a list of :class:`argusclient.model.Dashboard` objects with all fields populated.
+        """
+        assert ownerName, "Expected a owner name"
+        return self.argus._request("get", "dashboards", params=dict(ownerName=ownerName, shared=shared, limit=limit, version=version))
```
Comment (hdara-sfdc) [inline argusclient/client.py:387]: Shouldn't this be `owner`?
```suggestion
        return self.argus._request("get", "dashboards", params=dict(owner=ownerName, shared=shared, limit=limit, version=version))
```

**Review by hdara-sfdc — CHANGES_REQUESTED** [review]

File: argusclient/client.py (hunk 1 above)
Comment (j-ma-sf) [reply argusclient/client.py]: Oh, yes, good catch!

**Comment by j-ma-sf** [comment]
@haridsv  Also updated the version

**Review by hdara-sfdc — APPROVED** [review]
lgtm

**Comment by j-ma-sf** [comment]
@haridsv Thanks!

**Pull request merged**